### PR TITLE
No mem for samtools

### PR DESCRIPTION
**Pull request by alneberg**
Mostly for testing.

## PR checklist
 - [ ] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/sarek branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/sarek)
 - [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).
 - [ ] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** [guidelines](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -677,7 +677,7 @@ process MapReads {
         ${convertToFastq}
         bwa mem -K 100000000 -R \"${readGroup}\" ${extra} -t ${task.cpus} -M ${fasta} \
         ${input} | \
-        samtools sort --threads ${task.cpus} -m 2G - > ${idSample}_${idRun}.bam
+        samtools sort --threads 8 - > ${idSample}_${idRun}.bam
```
Comment (MaxUlysse) [inline main.nf]: Remind me why we're requesting 8 here?
won't there be an issue on smaller settings like travis/github actions?

Comment (alneberg) [reply main.nf]: It was mostly to mimic @drpatelh label `cpus_8`

Comment (alneberg) [reply main.nf]: We can perhaps think of a different way of doing it in the production code

Comment (MaxUlysse) [reply main.nf]: Let's see if Travis is failing or not, before fixing it ;-)

**Pull request closed without merging**